### PR TITLE
chore: Sends automatic slack message to docs-oncall when review is required in PR 

### PR DESCRIPTION
**Pull request by lantoli**
## Description

Sends automatic slack message to docs-oncall when review is required in PR.

Slack message is sent if PR goes to ready for review and there are changes in changelog entries or documentation (resource / data source guides and migration guides)

Link to any related issue(s): CLOUDP-242971

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR. A migration guide must be created or updated if the new feature will go in a major version.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR. A migration guide must be created or updated.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contributing guides](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/README.md)
- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [x] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
- [x] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

File: website/docs/hello.txt
Comment (lantoli) [inline website/docs/hello.txt:1]: this is to trigger the slack notification, will delete before merging

**Review by blva — APPROVED** [review]
🚀

File: .github/workflows/notify-docs-team.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+        with:
+          payload: |
+            {
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "hey @cloud-docs-on-call, this is :apix: bot, can you please review Terraform <${{ github.event.pull_request.html_url }}|PR ${{ github.event.pull_request.number }}>? thanks a lot!"
```
Comment (oarbusi) [inline .github/workflows/notify-docs-team.yml]: in other slack notifications we set the tag to the oncall as a secret to not show it in the public repo ([link](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2052#discussion_r1535655077)). Would probably do the same here

**Review by oarbusi — APPROVED** [review]

File: .github/workflows/notify-docs-team.yml (hunk 1 above)
Comment (lantoli) [reply .github/workflows/notify-docs-team.yml]: ok, will change

**Pull request merged**